client/src/components/polls/PollModal.js

```diff
@@ -24,12 +42,6 @@ export default function PollModal(props) {
       case 'title':
         setTitle(e.target.value);
         break;
-      case 'image1':
```

Comment:
I think you previously had more cases and removed them, but now you have a switch statement with just one case